common/src/main/scala/explore/HelpContext.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore
+
+import cats.syntax.all.*
+import crystal.react.View
+import crystal.react.hooks.*
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.string.*
+import explore.model.Help
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.feature.Context
+import japgolly.scalajs.react.vdom.html_<^.*
+import monocle.Focus
+import org.http4s.Uri
+import org.http4s.implicits.*
+
+case class HelpContext(
+  rawUrl:        Uri,
+  editUrl:       Uri,
+  user:          String,
+  project:       String,
+  displayedHelp: View[Option[Help.Id]]
+)
+
+object HelpContext:
+  val ctx: Context[HelpContext] = React.createContext(null) // No default value
```

Comment:
So this becomes available for use e.g. in other hooks?

Reply:
Yup, if you `.useContext(HelpContext.ctx)` then this will be available as another parameter in subsequent hooks (and render).